04-cond.Rmd

```diff
@@ -212,6 +212,38 @@ In this case, "either" means "either or both", not "either one or the other but
 > plot_dist(dat[1:5, 10], threshold = 10)                    # samples (rows) 1-5 on day (column) 10
 > ```
 
+
+> ## Challenge - Putting together data analysis, conditionals, and plotting   {.challenge}
+>   + For this challenge, fill in the blanks of the function hist_max() below to create a function that does the following:  
+  
+> ```{r, eval=FALSE}
```

Comment:
Don't put the challenge description in a code block. Instead start each line with `>`.